### PR TITLE
Made the blogs section responsive while keeping initial design.

### PR DESCRIPTION
**Pull request by Aryan3212**
I kept the 3-column layout for large screens and reduced column number as screen decreases on the blogs page. 

>Laptop and above:
image

>Tablet:
image

>Mobile:
image

### CODE REVIEW

**Comment by Aryan3212** [comment]
There might be some issues with code styling. I can fix those and re-commit.

File: pages/blog/index.js [hunk 1]
```diff
@@ -8,64 +8,73 @@ import { useIsomorphicLayoutEffect } from "../../utils";
...
+        <>
+            <Head>
+                <title>Blog</title>
+            </Head>
+            <div className="container mx-auto mb-10">
+                <Header isBlog={true}></Header>
+                <div className="mt-10">
+                    <h1
+                        ref={text}
+                        className="mx-auto text-6xl mob:text-6xl laptop:text-8xl mob:p-2 text-bold w-full mob:w-full laptop:w-4/5"
```
[the remark this answers is not in the document]
Comment (Aryan3212) [reply pages/blog/index.js:26]: Yup that makes sense, I'll work on that.

**Review by Aryan3212 — COMMENTED** [review]
I've made the changes mentioned. Please have a look.

**Review by chetanverma16 — APPROVED** [review]

**Comment by chetanverma16** [comment]
Nice Work!!

**Pull request merged**